docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -235,28 +235,28 @@ https://kafka.apache.org/documentation/#security_authz_primitives[Authorization
 In {product-kafka}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
 
 .Prerequisites
-* You've created a Kafka instance and the instance is in *Ready* state.
+* You've created a Kafka instance and the instance is in the *Ready* state.
 * The user account or service account that you're setting permissions for has been created in the organization.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. On the {service-url-kafka}[_Kafka Instances_^] page of the {product-kafka} web console, click the name of the Kafka instance that you want to set permissions for.
```

Comment:
I don't think we want the underscores around 'Kafka Instances'.

Reply:
Apologies for all the rogue underscores- copy and paste job gone bad!